OpenRA.Mods.Common/Widgets/Logic/Installation/DownloadPackageLogic.cs

```diff
@@ -208,6 +208,7 @@ void ShowDownloadDialog()
 					if (i.Error != null)
 					{
 						onError(Download.FormatErrorMessage(i.Error));
+						cancelButton.OnClick = Ui.CloseWindow;
```

Comment:
Hmm... .Wouldn't it make more sense to add this line to the error handler action instead (somewhere around line 115)? Alternatively, you'd have to add this same line before line 131.